client/tsconfig.json

```diff
@@ -15,8 +15,8 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
-    "jsx": "preserve"
+    "jsx": "preserve",
+    "isolatedModules": true
```

Comment:
does this ordering impact any functionality? or should we just sort these properties alphabetically?

Reply:
Changed fields of `tsconfig.json` to be in sorted order.